marathon.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "hdfs",
+    "cmd": "cd hdfs* && ./bin/hdfs-mesos",
+    "uris": [
+        ""
```

Comment:
@abhay-agarwal @elingg This shouldn't be done with a tarball. The build of the docker image should make the tarball. It should just be part of the image.